check_source.py

```diff
@@ -48,6 +49,11 @@ def target_project_config(self, project):
         self.repo_checker = config.get('repo-checker')
         self.devel_whitelist = config.get('devel-whitelist', '').split()
         self.skip_add_reviews = False
+        self.bad_rpmlint_entries = [
+            'polkit-unauthorized-',
+            'polkit-untracked-',
+            'permissions-',
+        ]
```

Comment:
If these are not configurable they should be a constant. It seems like they auto to just be configurable as Factory will be the only user and thus the whole list can be managed in Factory remote config (`OSRT:Config` attribute on `openSUSE:Factory`).

Reply:
Also as suggested by @lnussel this should probably also handle `.*` filters were possible.